lessons/Early-Math/evaluating-exponents/langs/en-us/index.adoc

```diff
@@ -0,0 +1,183 @@
+= Evaluating Exponents
+
+@description{Students use Circles of Evaluation analyze the structure of expressions that include exponents.}
+
+@lesson-prereqs{absolute-value-and-opposites-2}
+
+@keywords{exponent, base}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Translate between words, mathematical expressions, and Circles of Evaluation.
+* Evaluate expressions involving exponents and one other operation.
+* Evaluate expressions involving exponents and negations.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to analyze the structure of expressions with exponents.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* Ensure that students are comfortable evaluating exponents during the Launch. Circles of Evaluation will help students parse expressions that involve exponents.
+
+|===
+
+== Expressions with Exponents and One Operation @duration{30 minutes}
+
+=== Overview
+
+Sudents use Circles, words, and mathematical notation to analyze the structure of expressions with exponents and one operation.
+
+=== Launch
+
+[.lesson-instruction]
+- Show students the following expression: @math{2^3}
+- One way of reading this expression is "two raised to the power of three." Do you know any other ways of translating this expression into words?
+** _Two to the third power; two to the third; two cubed_
+- What does it mean to _cube_ a number?
+** We multiply it by itself three times (literally, compouting the volume of a cube with a side-length of that number)
```

Comment:
I love that you take this there... and... I think if you're going to go there, it deserves a visual of a cube with all 3 dimensions labeled as being 2in or 2cm.